index.bs

```diff
@@ -2985,14 +2986,14 @@ To <dfn>add or put</dfn> with |handle|, |value|, |key|, and |no-overwrite flag|,
 
 1. Return the result (an {{IDBRequest}}) of running
     [=asynchronously execute a request=]
-    with |handle| as |source| and
+    with |handle| as <var ignore>source</var> and
     [=store a record into an object store=] as
-    |operation|, using |store|, the |clone| as |value|, |key|, and
+    <var ignore>operation</var>, using |store|, the |clone| as <var ignore>value</var>, |key|, and
     |no-overwrite flag|.
```

Comment:
This is pretty confusing, as it mixes named and positional parameters. I'd suggest just using "_store_, _value_, _key_, and _no-overwrite flag_".

Reply:
Done!